cluster-autoscaler/utils/nodegroupset/compare_nodegroups.go

```diff
@@ -34,7 +34,7 @@ const (
 	MaxFreeDifferenceRatio = 0.05
 	// MaxMemoryDifferenceInKiloBytes describes how much memory
 	// capacity can differ but still be considered equal.
-	MaxMemoryDifferenceInKiloBytes = 128000
+	MaxMemoryDifferenceInKiloBytes = 256000000
```

Comment:
With the experiments with m5.large machines, I saw the following allocatable for 3 nodes with the same machine-type. 
```
{
  "cpu": "2",
  "ephemeral-storage": "50553132Ki",
  "hugepages-1Gi": "0",
  "hugepages-2Mi": "0",
  "memory": "7855172Ki",
  "pods": "110"
}
{
  "cpu": "2",
  "ephemeral-storage": "50553132Ki",
  "hugepages-1Gi": "0",
  "hugepages-2Mi": "0",
  "memory": "7941188Ki",
  "pods": "110"
}
{
  "cpu": "2",
  "ephemeral-storage": "50553132Ki",
  "hugepages-1Gi": "0",
  "hugepages-2Mi": "0",
  "memory": "7855172Ki",
  "pods": "110"
}
```

And code-wise the difference between allocatable.memory for second and third node came out to be: `8.8072192e+07` . This confuses a bit, but autoscaler couldn't consider the 2nd and 3nd node-groups similar, and couldn't balance. I'll take a defensive approach and have a large value to balance the machines.

Reply:
https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/processors/nodegroupset/compare_nodegroups.go#L66:1 upstream seems to have fixed this though.

Reply:
I think there are good changes in upstream in terms of comparing labels, better if we fix it while rebasing. Current changes seem to be fine for now.

Reply:
Okay.